bugsnag-plugin-android-ndk/src/main/java/com/bugsnag/android/NdkPlugin.kt

```diff
@@ -39,4 +42,8 @@ internal class NdkPlugin : Plugin {
     }
 
     override fun unload() = disableCrashReporting()
+
+    fun getSignalStackTrace(info: Long, userContext: Long): List<Stackframe> {
+        return nativeBridge.getSignalStackTrace(info, userContext)
```

Comment:
It won't let me put a null check here:

```
e: warnings found and -Werror specified
w: /Users/karl/work/code/bugsnag-android/bugsnag-plugin-android-ndk/src/main/java/com/bugsnag/android/NdkPlugin.kt: (48, 13): Condition 'nativeBridge == null' is always 'false'
```

Reply:
That'll be because the `nativeBridge` property was switched to `lateinit var` - my bad. If we just switch it back to `var nativeBridge: NativeBridge?` then the null check will be possible.